phpBB/phpbb/db/migration/schema_generator.php

```diff
@@ -59,7 +61,15 @@ class schema_generator
 	 * @param string $table_prefix
 	 * @param array $tables
 	 */
-	public function __construct(array $class_names, \phpbb\config\config $config, \phpbb\db\driver\driver_interface $db, \phpbb\db\tools\tools_interface $db_tools, $phpbb_root_path, $php_ext, $table_prefix, $tables)
+	public function __construct(
+		array $class_names,
+		\phpbb\config\config $config,
+		\phpbb\db\driver\driver_interface $db,
+		\phpbb\db\tools\tools_interface $db_tools,
+		string $phpbb_root_path,
+		string $php_ext,
+		string $table_prefix,
+		array $tables)
```

Comment:
The closing parenthesis would look better on a new line.

Reply:
It looks okay to me. We also use this convention all over the codebase.